testutil/keeper/crosschain.go

```diff
@@ -174,6 +174,13 @@ func CrosschainKeeperWithMocks(
 		lightclientKeeper,
 	)
 
+	cctxGateways := map[chains.CCTXGateway]keeper.CCTXGateway{
```

Comment:
Just thinking about it, we can think about create a mock gateway in the future and make some of the test more like unit tests